publications-2007.bib

```diff
@@ -0,0 +1,408 @@
+   journal = "J. Inverse and Ill-Posed Problems",
+   year    = "2007",
+   volume  = "5",
+   pages   = "19--36",
+   url     = "",
+   doi     = ""
+}
+
+@mastersthesis{2007_30,
+   author  = "L. R&ouml;he",
+   title   = "Residuale Stabilisierung f&uuml;r Finite Elemente Verfahren bei inkompressiblen Str&ouml;mungen",
+   year    = "2007",
+   school  = "University of G&ouml;ttingen, Germany",
+   url     = ""
+}
+
+@phdthesis{2007_31,
+   author  = "M. Schmidt",
+   title   = "Systematic discretization of input/output maps and other contributions to the control of distributed parameter systems",
+   year    = "2007",
+   school  = "TU Berlin, Germany",
+   url     = ""
+}
+
+@article{2007_32,
+   author  = "R. B. Schulz and M. Schweiger and C. D'Andrea and G. Valentini and J. Peter and R. Cubeddu and S. Arridge and W. Semmler",
+   title   = "Applying Time-Dependent Data for Fluorescence Tomography",
+   journal = "European Conference on Biomedical Optics, Munich Germany",
+   year    = "2007",
+   volume  = "",
+   pages   = "",
+   url     = "https://www.osapublishing.org/abstract.cfm?uri=ECBO-2007-6626_19",
+   doi     = "10.1364/ECBO.2007.6626_19"
+}
+
+@article{2007_33,
+   author  = "M. Secanell and B. Carnes and A. Suleman and N. Djilali",
+   title   = "Numerical optimization of proton exchange membrane fuel cell cathodes",
+   journal = "Electrochimica Acta",
+   year    = "2007",
+   volume  = "52",
+   pages   = "2668--2682",
+   url     = "",
+   doi     = ""
+}
+
+@article{2007_34,
+   author  = "M. Secanell and K. Karan and A. Suleman and N. Djilali",
+   title   = "Multi-variable optimization of PEMFC cathodes using an agglomerate model",
+   journal = "Electrochimica Acta",
+   year    = "2007",
+   volume  = "52",
+   pages   = "6318--6337",
+   url     = "",
+   doi     = ""
+}
+
+@article{2007_35,
+   author  = "M. Steigemann and M. Fulland",
+   title   = "On the computation of the pure Neumann problem in 2-dimensional elasticity",
+   journal = "International Journal of Fracture",
+   year    = "2007",
+   volume  = "146",
+   pages   = "265--277",
+   url     = "",
+   doi     = ""
+}
+
+@article{2007_36,
+   author  = "M. Steigemann and M. Fulland and M. Specovius-Neugebauer and H. A. Richard",
+   title   = "Simulation of crack paths in functionally graded materials",
+   journal = "PAMM",
+   year    = "2007",
+   volume  = "7",
+   pages   = "4030029--4030030",
+   url     = "",
+   doi     = ""
+}
+
+@article{2007_37,
+   author  = "R. Ziegler and T. Nielsen and T. Koehler and D. Grosenick and O. Steinkellner and A. Hagen and R. Macdonald and H. Rinneberg",
+   title   = "Reconstruction of absorption and fluorescence contrast for scanning time-domain fluorescence mammography",
+   journal = "Optical Tomography and Spectroscopy of Tissue VII, Proc. SPIE 64340H/1-11",
+   year    = "2007",
+   volume  = "6434",
+   pages   = "",
+   url     = "",
+   doi     = ""
+}
+
+@phdthesis{2007_64,
```

Comment:
@bangerth In the original `publications.bib` that you gave me, this entry was among some listed for 2013. I think that's why its key index is 64, while the previous entry's is only 37.